frontend/src/__tests__/cypress/cypress/e2e/pipelines/Pipelines.cy.ts

```diff
@@ -116,9 +116,172 @@ describe('Pipelines', () => {
     cy.wait('@refreshVersions');
 
     // Verify the uploaded pipeline version is in the table
-    pipelinesTable.toggleExpandRowByIndex(0);
+    pipelinesTable.toggleExpandRowByName(initialMockPipeline.name);
     pipelinesTable.findRowByName('New pipeline version');
   });
+
+  it('delete single pipeline', () => {
+    let deleted = false;
+
+    initIntercepts();
+    initDeletePipelineIntercept(initialMockPipeline.id);
+    cy.intercept(
+      {
+        pathname: '/api/proxy/apis/v1beta1/pipelines',
```

Comment:
Should include the method property here and anywhere else it might be absent in the intercepts in this file for specificity.